homework.cls

```diff
@@ -138,47 +157,44 @@
 % Mathematics packages
 \RequirePackage{amsmath, amsthm, amssymb}
 \RequirePackage{mathtools}
+\RequirePackage{physics}
```

Comment:
Not sure what macros this exposes. As long as it doesn't override any of the `ams*` or `mathtools` macros, then it should be fine.

Reply:
Again, didn't mean to propose this, but [this package](https://www.ctan.org/pkg/physics) does provide a lot of handy math macros that I use all the time. For example you can easily do Dirac bra-ket notation like `\braket{\phi}{\psi}` and math things like `\norm{X}` for `||X||`.